src/pipecat/services/ai_services.py

```diff
@@ -186,6 +186,7 @@ async def run_tts(self, text: str) -> AsyncGenerator[Frame, None]:
 
     async def say(self, text: str):
         await self.process_frame(TextFrame(text=text), FrameDirection.DOWNSTREAM)
+        await self.flush_audio()
```

Comment:
`flush_audio` is only available in `AsynTTSService` since I thought it only made sense there. We have two options:

- Add `flush_audio` to `TTSService`
- Override `say()` in `AsyncTTSService` with

```
async def say(self, text: str):
    super().say(text)
    await self.flush_audio()
```

Reply:
I don't think there's anything to flush in `TTSService`, so maybe option 2 is better?